package.json

```diff
@@ -38,7 +40,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test --coverage",
```

Comment:
If you want to change the script so a to use npm test, then add  `-- -u --coverage` to ensure you update the snapshot everytime you run the tests.

Reply:
It is bringing this error when I add it to the package.json
![image](https://user-images.githubusercontent.com/29709981/51595268-661e3a80-1f07-11e9-8fc6-a22528dccb7c.png)